Directory.Build.props

```diff
@@ -7,14 +7,15 @@
     <ConfigPathSegment>Debug</ConfigPathSegment>
     <ConfigPathSegment Condition="'$(Configuration)'!=''">$(Configuration)</ConfigPathSegment>
     <LangVersion>11.0</LangVersion>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
```

Comment:
This property is declared in the Directory.Packages.props too. Is it necessary to declare it here ?

Reply:
Good catch. I'll remove it from here.